src/monopoly/board/monopoly_board.js

```diff
@@ -0,0 +1,43 @@
+var TableTop = require('../../../tabletop/tabletop');
+var _ = require('lodash');
+var inherits = require('util').inherits;
+
+// more of a placeholder for now -
+// should talk when merged to properly integrate this with
+// existing "board" and "players" implementations if any exist
+function MonopolyBoard() {
+    TableTop.Board.call(this);
+    this.tiles = [];
+}
+
+inherits(MonopolyBoard, TableTop.Board);
+
+MonopolyBoard.prototype.findTileForToken = function(token) {
+    var tile = _.find(this.tiles, function (n) {
+        return (n.tokens.indexOf(token) != -1);
+    });
+
+    return tile;
+};
+
+MonopolyBoard.prototype.moveTokenToTile = function(token, tile) {
+    var oldTile = this.findTileForToken(token);
+    if (oldTile != undefined) {
+        oldTile.tokens.splice(this.tiles.indexOf(token), 1);
+    }
+
+    tile.addToken(token);
+};
+
+MonopolyBoard.prototype.destroyToken = function(token) {
```

Comment:
I was wondering the same thing, but opted to keep at Monopoly for now cause I was trying to focus on the drills. I don't think in go in TableTop.Board because it doesn't work on all board types, so we're going to have to subclass to PathBoard to include this at the framework level.  Currently the moveTokenToTile, findTileForToken, etc. are required to be implemented by the subclasses of board. Should I go ahead and make this change before submission?